schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+                    "type": "boolean"
+                },
+                "webByVersion": {
+                    "description": "Show version as prefix in web adapter (usually - ip:port/material, webByVersion - ip:port/1.2.3/material)",
+                    "type": "boolean"
+                },
+                "webExtendable": {
+                    "description": "If web server in this adapter can be extended with plugin/extensions like proxy, simple-api",
+                    "type": "boolean"
+                },
+                "webExtension": {
+                    "description": "Relative filename to connect the web extension. E.g. in simple-api \"lib/simpleapi.js\" relative to the adapter root directory. Additionally is native.webInstance required to say where this extension will be included. Empty means, it must run as own web service. \"*\" means every web server must include it.",
+                    "type": "string"
+                },
+                "webPreSettings": {
+                    "description": "Object of parameters that must be included into info.js by webServer adapter. (Example material)",
+                    "type": "object"
+                },
+                "webservers": {
+                    "description": "Array of web server's instances that should serve content from the adapters www folder",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "welcomeScreen": {
+                    "description": "Array of pages, that should be shown on the \"web\" index.html page. [\"vis/edit.html\", \"vis/index.html\"] or [{\"link\": \"vis/edit.html\", \"name\": \"Vis editor\", \"img\": \"vis/img/edit.png\", \"color\": \"blue\"}, \"vis/index.html\"]",
+                    "type": "array",
+                    "items": {
+                        "type": ["string", "object"]
+                    }
+                },
+                "welcomeScreenPro": {
+                    "description": "Array of pages used on access via ioBroker cloud, that should be shown on the \"web\" index.html page. [\"vis/edit.html\", \"vis/index.html\"] or [{\"link\": \"vis/edit.html\", \"name\": \"Vis editor\", \"img\": \"vis/img/edit.png\", \"color\": \"blue\"}, \"vis/index.html\"]",
+                    "type": "array",
+                    "items": {
+                        "type": ["string", "object"]
+                    }
+                },
+                "wwwDontUpload": {
+                    "description": "Do not upload the www directory into DB. Used only for admin.",
+                    "type": "boolean"
+                }
+            }
+        },
+        "objects": {
+            "description": "Objects which will be created for the adapter",
+            "type": "array",
+            "items": {
+                "type": "object"
```

Comment:
We could enforce some properties, but I'm not sure if that is beneficial

Reply:
maybe `_id`, onyl thing which has to be there